pages/data.py

```diff
@@ -4,7 +4,7 @@
 The workaround is to check if the input value is None.
 """
 from dash import dcc, html, Input, Output, callback, register_page, dash_table
-
+from datetime import date
```

Comment:
I am merging this for now, but we should revisit the time handling in general since datetime does not handle timezones very well. We should use `arrow` like the rest of the e-mission codebase, and also consider date formatting etc.

Can you please file a follow-up issue for this?